### ISSUE TITLE
'Tensor' object has no attribute 'size'

### ISSUE THREAD

**Issue by waqarkaleemkhan** [huggingface/transformers#11348]
I am trying to implement a transformer for language classification using TensorFlow below is the model code I used but it throws an error of tensor object has no attribute size please help for the notebook you can visit the below link

https://github.com/waqarkaleemkhan/Transformer_for_Language_classification/blob/master/Transformer_for_language_classification/model.ipynb

from transformers import AutoModel
bert = AutoModel.from_pretrained('bert-base-cased')
input_ids = tf.keras.layers.Input(shape=(SEQ_LEN,), name='input_ids', dtype='int32')
mask = tf.keras.layers.Input(shape=(SEQ_LEN,), name='attention_mask', dtype='int32')
embeddings = bert(input_ids, attention_mask=mask)
X = tf.keras.layers.LSTM(64)(embeddings)
X = tf.keras.layers.BatchNormalization()(X)
X = tf.keras.layers.Dense(64, activation='relu')(X)
X = tf.keras.layers.Dropout(0.1)(X)
y = tf.keras.layers.Dense(3, activation='softmax', name='outputs')(X)

**Comment by LysandreJik** [comment]
Hi! You're using `AutoModel`, which is a pytorch model, with TensorFlow instructions. Change to `TFAutoModel`!

**Comment by waqarkaleemkhan** [comment]
Hi @LysandreJik Thanks for your response when I try to import TFAutoModel from TensorFlow it gives an error that cannot import name 'TFAutomodel' from 'transformers' (unknown location)
my environment 
python =3.7
TensorFlow= 2.0
can you please guide me further what to do

**Comment by LysandreJik** [comment]
What's your `transformers` version? Can you install a more recent version of `tensorflow` and see if it fixes your issue?

**Comment by waqarkaleemkhan** [comment]
I have updated my TensorFlow and transformer but still I have the same issue
